lib/node/nodes/data-observatory-multiple-measures.js

```diff
@@ -123,7 +125,7 @@ var queryTemplate = Node.template([
     '_data AS ( ',
     '  SELECT id AS __obs_id__, {{=it.obsColumns}}',
     '  FROM cdb_dataservices_client._OBS_GetData_exception_safe(',
-    '    (SELECT ARRAY_AGG((the_geom, cartodb_id)::geomval) FROM _source),',
+    '    (SELECT geomvals FROM _summary),',
```

Comment:
When I first saw it I thought we were just sending the summary to get the values, then I thought we were sending all the geoms to get the metadata and actually it is neither of those. Took me a third look to figure out what it was actually doing.

The potential problem is naming "summary" to something else. Confronted to the API docs confuses anyone coming after, as happened to dgaubert and myself. IMHO, naming is important enough to change it in master, even if you don't publish a new version.